app/src/views/Cart.vue

```diff
@@ -86,8 +86,15 @@
               <div class="col-span-4 lg:col-span-1">
                 <div class="text-grey-400 text-left lg:text-right">
                   <!-- TODO use real match estimates -->
-                  <p v-if="true">not in an active round</p>
-                  <p v-else>USD estimated matching</p>
+                  <p v-if="clrPredictions[item.grantId]">
+                    <span v-for="(clr, index) in clrPredictions[item.grantId]" :key="index">
+                      {{ formatNumber(clr.matching, 2) }} {{ clr.matchingToken.symbol }}
+                      {{ index !== clrPredictions[item.grantId].length - 1 ? '+' : '' }}
+                      <br />
+                      estimated matching
```

Comment:
Why do some of these have match amounts of zero? 

![image](https://user-images.githubusercontent.com/17163988/133265269-5c8af39c-df1e-471b-a780-7bb3a4b7ca7c.png)

Reply:
Hmmm, I'm not sure, could you try it with the latest changes?